lib/rules/at-rule-empty-line-before/README.md

```diff
@@ -77,15 +87,25 @@ a {}
 
 ## Optional secondary options
 
-### `except: ["after-same-name", "inside-block", "blockless-after-same-name-blockless", "blockless-after-blockless", "first-nested"]`
+### `except`
+
+```json
+{ "except": ["array", "of", "options"] }
+```
```

Comment:
👍 These changes really clean up the sidebar on the website.

Reply:
The sidebar should show nested headers.
i.e. in this case `"after-same-name"`, `"inside-block"`, etc.

This could be discussed in a new enhancement issue.

Reply:
@Mouvedia It's a matter on the website, not these documents in the stylelint repo. So, if you want, please open an issue on the stylelint.io repo.

We can configure the heading level displayed via the Docusaurus config:
- https://docusaurus.io/docs/api/themes/configuration#table-of-contents
- https://github.com/stylelint/stylelint.io/blob/71c8a2d158fb4ec2ca2f82b2e47e9fec092f4dfb/docusaurus.config.js#L37

I guess the default `minHeadingLevel: 2` is good for now and increasing it to `3` might make the right sidebar a bit messier. But we could discuss it in a new issue.